libs/blocks/aside/aside.css

```diff
@@ -480,6 +480,14 @@
   height: var(--icon-size-m);
 }
 
+.aside.promobar .promo-text[data-align="center"] {
```

Comment:
Just so you are aware, we saw issues copying these Word data attributes from the sidekick library. I am not sure if that is still a problem, but we had to go with `center` as a block variant as a result. More context [here](https://github.com/adobecom/milo/pull/627).

Reply:
Thanks for the context, Sean! I'm not sure this would be copied over using the Library, but it also doesn't hurt to have it. I'll look into adding the `center` variant for the `promobar` variant as well.

Reply:
Adding the `center` variation skews the layout because of https://github.com/adobecom/milo/blob/main/libs/blocks/aside/aside.css#L48-L50 interacting with https://github.com/adobecom/milo/blob/main/libs/blocks/aside/aside.css#L634-L636.

I'm not aware of all the Aside use-cases, as it seems highly complex, so I think I'll stay away from additional logic for the moment. One could argue that if `:not(.center)` is used on L634 (`.aside:not(.notification):not(.inline):not(.center) .foreground.container> :last-child`), then it could also be used on L48 (`.aside:not(.notification):not(.inline) .foreground> :last-child`), which would fix the layout issues I'm seeing, but I wouldn't want to make assumptions.